spec/model/vm_host_spec.rb

```diff
@@ -249,7 +249,9 @@
   end
 
   it "initiates a new health monitor session" do
-    expect(vh.sshable).to receive(:start_fresh_session)
+    sshable = instance_double(Sshable)
+    expect(vh).to receive(:sshable).and_return(sshable)
+    expect(sshable).to receive(:start_fresh_session)
```

Comment:
```suggestion
    sshable = instance_double(Sshable, start_fresh_session: nil)
    expect(vh).to receive(:sshable).and_return(sshable)
```

Ever so slightly unorthodox, but shorter

Reply:
You get an extra invariant out of yours, so I decided I liked it better even if it costs a line.